tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -46,6 +46,15 @@ class GraphHelper {
 		'Denied' => '63e64e19-8d43-42ec-a738-2b6af2610efa',
 	];
 
+	public const PERMISSIONS_ROLES_MAPPINGS = [
+		'View' => 'view',
```

Comment:
i think this permission is dedicated for link share

![Screenshot from 2025-01-15 08-42-10](https://github.com/user-attachments/assets/8406f9ff-85d9-4190-b0ac-39ad7fd0d2e5)



```suggestion
	public const SHARING_LINK_TYPE_MAPPINGS = [
		'View' => 'view',
```